### ISSUE TITLE
EASYSSL_STATIC_SSL ?

### ISSUE THREAD

**Issue by jhay06** [QuasarApp/easyssl#6]
Does it need to have openssl library first ?

**Comment by EndrII** [comment]
Yes, but if it is trouble for you, you can prepare build openssl library with my build [script](https://github.com/QuasarApp/openssl-builder).

**Comment by EndrII** [comment]
@jhay06 but I can add new option for the preparing OpenSSL libraries before build easyssl. 
for example **BUILD_WITH_OPENSSL**

This will be support Linux, Android, iOS and Mac platforms (arm/x86) by default. But not windows.
